barchart/rsbuild.config.ts

```diff
@@ -56,6 +56,7 @@ export default defineConfig({
       },
       dts: false,
       runtime: false,
+      getPublicPath: `function() { const prefix = window.PERSES_PLUGIN_ASSETS_PATH || window.PERSES_APP_CONFIG.api_prefix || ""; return prefix + "${assetPrefix}"; }`,
```

Comment:
I updated the comment with a simplified version that accounts for missing app config:

```
getPublicPath: `function() { return (window.PERSES_PLUGIN_ASSETS_PATH || window.PERSES_APP_CONFIG?.api_prefix || "") + "${assetPrefix}"; }`,
```